lib/game.rb

```diff
@@ -1,35 +1,43 @@
-require_relative './player.rb'
-require_relative './board.rb'
 require_relative './ui.rb'
+require 'set'
 
 class Game
   include UI
-  def initialize(board,players)
+  def initialize(board, players)
     @board = board
     @players = players
-    output('Welcome to Tic-tac-toe!', true)
-    2.times { |n| get_player_data(n) }
+  end
+
+  WINNING_SETS = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7], [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]].freeze
+  private_constant :WINNING_SETS
+
+  def play
```

Comment:
Let's simplify this method. It's overcrowded with logic. At the very least some of the things it does could very well be their own methods that this play method just calls. Having it like this makes it so messy.